rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -1550,11 +1552,11 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
         Await.result(inspectTask.runToFuture, 3.seconds)
     }
     result.exprs should be(Seq(Expr(EListBody(EList(List())))))
-    errorLog.readAndClearErrorVector should be(Vector.empty[InterpreterError])
+    errorLog.readAndClearErrorVector.unsafeRunSync should be(Vector.empty[InterpreterError])
```

Comment:
That's why I love test code duplication so much ;/

I won't insist that you clean this up, but since 5 people seem to be engaged in this PR, I had to mention this :)